config/metrics.js

```diff
@@ -61,6 +61,15 @@ module.exports = {
 		extract: (item) => calcRestrictedSourcePercentage(getBPRules(item)),
 		permissions: ['ADMIN'],
 	},
+	RequiredDeploymentEnvironments: {
+		// ✓ when at least one required deployment environment exists, else ✗
+		extract(item) {
+			const envs = getBPRules(item)?.RequiredDeploymentEnvironments;
+			return !!envs && envs.length >= 0;
+		},
+		permissions: ['ADMIN'],
```

Comment:
does this work with `MAINTAIN` as well? or `WRITE`? reference other metrics and find out to make sure it works fine